site/content/en/docs/Guides/Client SDKs/unreal.md

```diff
@@ -75,4 +75,139 @@ Available settings:
 - Health Ping Seconds. Interval of the server sending a health ping to the Agones sidecar. (default: `5`)
 - Request Retry Limit. Maximum number of times a failed request to the Agones sidecar is retried. Health requests are not retried. (default: `30`)
 - Send Ready at Startup. Automatically send a Ready request when the server starts. Disable this to manually control when the game server should be marked as ready. (default: `true`)
+{{% /feature %}}
+{{% feature publishVersion="1.8.0" %}}
+# Agones Unreal SDK
+
+This is a SDK inspired by the REST API to the Agones sidecars that allows engineers to talk to the sidecar from either C++ or Blueprints.
```

Comment:
Idea: put this line right under `## Getting Started` as the first line - might make more sense contextually on the page?

Not required, just thought I'd suggest it, see if it worked for you as well.

Reply:
agree, moved this into getting started replacing the existing text.